### ISSUE TITLE
Tab managing

### ISSUE THREAD

**Issue by rayman89** [adamjamesadair/manage-my-tabs#130]
So far the tab manager only helps with filtering/finding tabs and closing them. Are there plans to include more features like manual re arrange of the tabs being able to select multiple tabs to close reload suspend bookmark copy their url move to a new window or existing window etc?

**Comment by rayman89** [comment]
Maybe this belongs on an other issue but this depends on this issue. If you are going to implement those tab managing features it would be nice to have a view that does not group tabs by url and shows the tabs like they appear on each window.

**Comment by adamjamesadair** [comment]
@rayman89 Thank you for the suggestions. You can see plans for upcoming releases here: https://github.com/adamjamesadair/manage-my-tabs/projects/4.
Some of the features you mentioned are open as issues #60 and #61. I've opened issues #131, #132 and #133 to address the other features.

**Comment by BorFour** [comment]
Yeah, I agree with @rayman89. It would be nice if the criteria used to group the tabs could be changed. But actually, if you go to the side bar and change the "Max number of tabs" to 1 and "Sort groups" to "Window order", you should be able to have them like that. A little bit tricky, but it can be done.

**Comment by rayman89** [comment]
@BorFour On the issues you listed you refer to actions that affect 1 tab at a time. Are you willing to implement doing stuff like moving to an other window re arranging bookmarking etc in bulk? (please allow to manually select the tabs not only do them in bulk when they are grouped).

**Comment by adamjamesadair** [comment]
@rayman89 #143 adds the option to reload or send individual tabs to a different window. With #146 we plan to extend that functionality to tab groups and windows. I've opened issue #147 to address selecting manually selecting tabs, tab groups and windows.
